getting-started/main.cpp

```diff
@@ -142,9 +142,9 @@ static void sign_a_message_using_rsa(void)
 {
     psa_status_t status;
     psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
-    uint8_t key[] = RSA_KEY;
+    static uint8_t key[] = RSA_KEY;
```

Comment:
Having a `static` variable is unusual, and usually done when the value needs to be preserved between calls, which is not the case here. Please add a comment in the code to indicate why these two variables are `static`.